pipelines/h6c/rtp/v2/stage_1_task_scripts/do_ALL_SKY_IMAGE_MOVIE.sh

```diff
@@ -0,0 +1,23 @@
+set -e
+
+#This script smashes a stack of jpgs into a movie. Used for widefield broadband RFI imaging.
+
+src_dir="$(dirname "$0")"
+source ${src_dir}/_common.sh
+
+# the only input we need is the filename
+fn=${1}
+CASA_ENV=${2}
+
+# Get the integer JD from filename
+jd=$(get_int_jd ${fn})
+#put all the jpgs into a sub-dir for easy stacking with a ffmpeg glob
+mkdir -p ${jd}_imgs
+mv *${jd}*jpg ${jd}_imgs
+
+conda activate $CASA_ENV
+#image stack is in its own dir because glob dereferencing is hard
+cd ${jd}_imgs
+ffmpeg -f image2 -pattern_type glob -i '*.jpg' HERA_${jd}_automatic.mp4
+cp  HERA_${jd}_automatic.mp4 ..
+# TODO put movie in Librarian and upload to enterprise for quick online viewing
```

Comment:
We should probably add the upload steps before putting this in the pipeline, no? Also, what's the plan if the link to site is down?

Reply:
Dang it, looks like I lost a line in the transition.
Current plan is to drop movies in a special dir on sn1. A regular (hourly?) will rsync to heranow.  This will ride out link outages as well as can be and get us our movies fast as possible.
TODO to get movies into librarian. Important but not essential for our urgent need now which needs publicly viewable and distributable files right away.

Reply:
Have added a line right after this that copies to /mnt/sn1/nightly_movies
TODO: add an rsync. I have prototyped a command but haven't added it to cron yet